### PR TITLE
brackets: 1.8 -> 1.9

### PR DESCRIPTION
**Pull request by ndowens**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/editors/brackets/default.nix [hunk 1]
```diff
@@ -9,55 +11,56 @@ let
...
-stdenv.mkDerivation rec {
-  name = "brackets-${version}";
-  version = "1.8";
-
-  src = fetchurl {
-    url = "https://github.com/adobe/brackets/releases/download/release-${version}/Brackets.Release.${version}.64-bit.deb";
-    sha256 = "0b2k0vv1qwmsg1wckp71yrb86zp8zisskmzzvx9ir19bma9jzr42";
-    name = "${name}.deb";
-  };
+  stdenv.mkDerivation rec {
```
Comment (joachifm) [inline pkgs/applications/editors/brackets/default.nix]: Looks like this adds an additional space to each line.

Comment (ndowens) [inline pkgs/applications/editors/brackets/default.nix]: Used G=gg in VIM to fix some indent issues I noticed

Comment (joachifm) [inline pkgs/applications/editors/brackets/default.nix]: I don't follow? The patch seems like it shifts everything to the right, including lines which should not be indented.

Comment (ndowens) [inline pkgs/applications/editors/brackets/default.nix]: Fixed it, seen what you were talking about; and also I do not know what is going on with git rebase with me lately, I can't squish the commits. I asked in IRC but no one answered

Comment (joachifm) [inline pkgs/applications/editors/brackets/default.nix]: I usually just rebase onto master (or whatever branch I'm targeting), as in `git fetch upstream master && git rebase -i upstream/master`. That's much easier than having to work out how many steps to jump back &c.

Comment (ndowens) [inline pkgs/applications/editors/brackets/default.nix]: THat doesnt work either

Comment (joachifm) [inline pkgs/applications/editors/brackets/default.nix]: Hm, and you're doing the rebase with the PR branch checked out? If so, and it my command doesn't work, then I have no idea ...

Comment (ndowens) [inline pkgs/applications/editors/brackets/default.nix]: @joachifm me neither; as I am in the branch and yours didnt work either; lately it has been doing that for other packages too

Comment (joachifm) [inline pkgs/applications/editors/brackets/default.nix]: The only explanation that makes sense to me is that you're rebasing onto the wrong tip somehow. For example, have you verified that the upstream remote points to NixOS/nixpkgs and not your own fork?

Comment (ndowens) [inline pkgs/applications/editors/brackets/default.nix]: origin  git@github.com:ndowens/nixpkgs.git (fetch)
origin  git@github.com:ndowens/nixpkgs.git (push)
upstream        https://github.com/nixos/nixpkgs (fetch)
upstream        https://github.com/nixos/nixpkgs (push)

Comment (joachifm) [inline pkgs/applications/editors/brackets/default.nix]: and `upstream/master` is synchronized with upstream? If so I literally have no idea how it is that rebasing onto it wouldn't work. It certainly does for me ...

Comment (ndowens) [inline pkgs/applications/editors/brackets/default.nix]: AFAIK it is, I keep my repo up2date from upstream too; so I am unsure what is going on :\

**Pull request closed without merging**